README.md

```diff
@@ -1,4 +1,47 @@
-# osl-resources
+# osl-resources Cookbook
 
-TODO: Enter the cookbook description here.
+This is the osl-resources cookbook for the OSL unmanaged and managed hosts.
 
+## Platforms
+
+The following platforms and versions are tested and supported using [test-kitchen](http://kitchen.ci/)
+
+- CentOS 7/8
+- Debian 10/11
+- Ubuntu 18.04 / 20.04
+
+## Resources
+
+- [osl_authorized_keys](https://github.com/osuosl-cookbooks/osl-resources/blob/REK/Initial_PR/documentation/osl_authorized_keys.md)
```

Comment:
These should be relative links, otherwise they will get broken due to branches:
```suggestion
- [osl_authorized_keys](documentation/osl_authorized_keys.md)
```